recipes/khiopsdriver-gcs/build.bat

```diff
@@ -0,0 +1,11 @@
+REM Echo all output
+@echo on
+
+REM Configure project
+cmake --fresh -G Ninja -D CMAKE_BUILD_TYPE=Release -B builds\conda -S .
+
+REM Build
+cmake --build builds\conda --parallel --target khiopsdriver_file_gcs
+
+REM Copy the lib for the driver package
+copy builds\conda\bin\khiopsdriver_file_gcs.dll %LIBRARY_PREFIX%\bin\libkhiopsdriver_file_gcs.dll
```

Comment:
Does `cmake --install builds\conda` not install the proper files.  If you do have to do this manually, also please make sure to copy `builds\conda\lib\khiopsdriver_file_gcs.lib` to `%LIBRARY_PREFIX%\lib`.  If this file is not generated automatically, please add `-DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON` to the Windows CMake call.

Reply:
Hello again, I just wanted to clarify the point regarding the "lib" file, in addition to the file rename operation explained above. Since this filesystem driver is exclusively used by the [khiops](https://www.khiops.org) program which uses dlopen/dlsym equivalents on all platforms to retrieve and use the provided functions, there is no need to include the lib file in the package.

Reply:
@conda-forge-admin, please ping conda-forge/help-c-cpp team

Reply:
Have you tried adding `-DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON` to see if it creates the file?  We prefer all of our shared libraries to generate this even if they are not currently used by any other packages because it leaves open the future possibility of other packages to link to it.  We may also implement automated auditing in the future of shared library packages to make sure they have the `.lib` files and we should future-proof here if possible.